coral-hive/build.gradle

```diff
@@ -28,12 +27,3 @@ generateGrammarSource {
     '-lib',
     'src/main/antlr/imports'
   ]}
-
-task customFatJar(type: Jar) {
```

Comment:
does this mean we dont need a fat-jar for coral-hive? 
how did we verify that there are no consumers of this jar?

Reply:
Thanks! I don't think it's used in Coral itself, and I don't know any consumers of it.
@wmoustafa do you remember the reason why you added this part of code?

Reply:
I think it helped testing/debugging on the cluster. I am okay removing it if preferred.